README.md

```diff
@@ -31,9 +31,11 @@ version=1.0.*
 2. Apply `org.shipkit.shipkit-auto-version` to the root project.
 
 3. Important! When using this plugin on CI make sure that your CI includes tags when making checkout.
-For example, when using GitHub actions, you need add something this:
+For example, the easiest way to achieve this with GitHub actions is to tweak 'fetch-depth':
 ```
```

Comment:
Out of the scope of changes, but we can use ```yaml here.

Reply:
Thanks! Updated.